OPERATOR.rst

```diff
@@ -156,28 +156,44 @@ label from blocking PR. Last, remove the blocking relationship.
 Upgrading GitLab
 ^^^^^^^^^^^^^^^^
 
-Occasionally it falls on the operator to upgrade the Azul GitLab instance. If
-the current major version is ``n`` and the latest available major version is
-greater than ``n+1`` (i.e. upgrading directly to the latest version would skip
-one or more major versions) then multiple successive upgrades must be made, such
-that no upgrade skips a major version. For example, if the current version is
-13.x.y and the latest available version is 15.x.y, then one would first upgrade
-to 14.x.y and then repeat the process to upgrade to 15.x.y.
-
-Before any changes are applied, stop the instance (do not terminate) and create
-a snapshot of its EBS volume. Edit ``terraform/gitlab/gitlab.tf.json.template.py``,
-updating the versions of the docker images for ``gitlab-ce`` and
-``gitlab-runner``. Then run::
+Operators must check for updates to GitLab on a daily basis. Updates to Gitlab
+can be quickly checked at the ``/help`` endpoint for `GitLab dev`_ and
+`GitLab prod`_. If there are updates available, find the new release for the
+`GitLab Docker image`_ and update the `Gitlab Terraform`_ file. When updating
+the GitLab instance check if there are applicable updates to the
+`GitLab runner image`_. Before starting the update process, check the applicable
+`GitLab release notes`_ for any upgrading instructions. When upgrading across
+multiple Gitlab versions, follow the prescribed GitLab `upgrade path`_.
+
+Before any changes are applied, have the lead check for any running
+``background migrations`` on the GitLab instances. Once all background
+migrations are complete, stop the instance (do not terminate) and create a
+snapshot of its EBS volume.
+
+Edit ``terraform/gitlab/gitlab.tf.json.template.py``, updating the versions of
+the docker images for ``gitlab-ce`` and ``gitlab-runner``. Then run::
 
     _select dev.gitlab
     cd terraform/gitlab
-    make apply
+    make
+
+The GitLab instance should be online again in 10 minutes or so. If it takes
+substantially longer, contact the lead. Once the ``dev`` GitLab instance has
+been successfully updated, the same changes need to be applied to the ``prod``
+GitLab instance. Once both instances are successfully updated, file a PR with
+the changes against the ``develop`` branch and request review from the lead.
 
 It may be necessary to set ``CI_COMMIT_REF_NAME=develop`` to work around
```

Comment:
``CI_COMMIT_REF_NAME=prod`` is needed for `prod`, even though the changes are based on `develop`.